src/Models/Sprint.php

```
@@ -137,4 +137,80 @@ public static function delete(Database $db, int $id): void
     {
         $db->query("DELETE FROM sprints WHERE id = :id", [':id' => $id]);
     }
+
+    // ===========================
+    // EXECUTION — START / CLOSE
+    // ===========================
+
+    /**
+     * Transition a sprint to active and record actual start time.
+     */
+    public static function startSprint(Database $db, int $id): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)